client/src/components/HeaderWithMenu/index.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+import { SearchOutlined, MenuOutlined, CloseOutlined } from '@ant-design/icons';
+import './style.css';
+import logo from '../../assets/images/logo.png';
+
+class HeaderWithMenu extends Component {
+  state = { show: false };
+
+  reverseShow = () => {
+    const { show } = this.state;
+    this.setState({ show: !show });
+  };
+
+  render() {
+    const { show } = this.state;
+    return (
+      <div>
```

Comment:
@MohammedAlghazali I think you don't need this div, the `main-header` div is already wrapping everything.

Reply:
oh, yes I made something and delete it and forget to remove this

Reply:
I divide the work to more than pull request